.github/ISSUE_TEMPLATE/config.yaml

```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: true
 contact_links:
   - name: 💬 Join Slack
-    url: "https://logfire.pydantic.dev/docs/help/#slack"
+    url: 'https://logfire.pydantic.dev/docs/help/#slack'
```

Comment:
The "Join Slack" is not present in this repository. This is the only difference between here and what is in the Logfire repository.